server/mergin/sync/public_api_controller.py

```diff
@@ -424,6 +424,37 @@ def get_project_by_uuid(project_id):  # noqa: E501
     return data, 200
 
 
+@auth_required
+def schedule_delete_project(project_id):  # noqa: E501
+    """Schedule deletion of the project.
+
+    :param project_id: Project id
+    :type project_id: str
+    """
+    project = require_project_by_uuid(project_id, ProjectPermissions.Delete)
+    project.removed_at = datetime.utcnow()
+    project.removed_by = current_user.username
+    project_deleted.send(project)
+    db.session.commit()
+
+    return NoContent, 204
+
+
+@auth_required
+def delete_project_now(project_id):  # noqa: E501
+    """Delete the project immediately.
+
+    :param project_id: Project id
+    :type project_id: str
+    """
+    project = require_project_by_uuid(project_id, ProjectPermissions.Delete)
```

Comment:
This means I can not rush delete once it is scheduled for deletion, right? Just let's make sure it matches requirements.